cluster/src/main/java/io/scalecube/cluster/ClusterConfig.java

```diff
@@ -105,6 +105,36 @@ public static Builder builder() {
     return new Builder();
   }
 
+  /**
+   * Creates a new {@link ClusterConfig.Builder} by the given {@link ClusterConfig}.
+   *
+   * @param clusterConfig cluster config
+   * @return {@link ClusterConfig.Builder}
+   */
+  public static Builder from(ClusterConfig clusterConfig) {
+    return new Builder()
+        .seedMembers(clusterConfig.seedMembers)
+        .metadata(clusterConfig.metadata)
+        .syncInterval(clusterConfig.syncInterval)
+        .syncTimeout(clusterConfig.syncTimeout)
+        .suspicionMult(clusterConfig.suspicionMult)
+        .syncGroup(clusterConfig.syncGroup)
+        .metadataTimeout(clusterConfig.metadataTimeout)
+        .pingInterval(clusterConfig.pingInterval)
+        .pingTimeout(clusterConfig.pingTimeout)
+        .pingReqMembers(clusterConfig.pingReqMembers)
+        .gossipInterval(clusterConfig.gossipInterval)
+        .gossipFanout(clusterConfig.gossipFanout)
+        .gossipRepeatMult(clusterConfig.gossipRepeatMult)
+        .port(clusterConfig.transportConfig.getPort())
+        .connectTimeout(clusterConfig.transportConfig.getConnectTimeout())
+        .useNetworkEmulator(clusterConfig.transportConfig.isUseNetworkEmulator())
+        .messageCodec(clusterConfig.transportConfig.getMessageCodec())
+        .maxFrameLength(clusterConfig.transportConfig.getMaxFrameLength())
+        .memberHost(clusterConfig.memberHost)
+        .memberPort(clusterConfig.memberPort);
+  }
```

Comment:
`transportConfig`  has  its own  `copy` or copyFrom`;  hence lines  129 .. 133  can be merged to one.

Reply:
done